decentraland/unity-explorer
# style: make avatar model x position responsive

## What does this PR change?
This PR was created to fix the anchor position of the avatar model, which depending on the screen resolution turns wrong giving as result is a lot of bottom margin as in the following example:

https://github.com/user-attachments/assets/8aadfa9b-6908-441b-98a3-3fdd14df5ef4

With this fix, the avatar is always anchored at the left center of the screen
https://github.com/user-attachments/assets/32067437-496d-4d3f-80cb-7d4fd466a659

### What's pending?
Scale the avatar model for wide screens to avoid it of being chopped at the top and bottom.

## How to test the changes?
1. Launch the explorer
2. Play with different screens resolutions on the authentication screen to check the avatar position is now constant.
